modules/effects/spec/integration.spec.ts

```diff
@@ -215,6 +216,60 @@ describe('NgRx Effects Integration spec', () => {
       // ngrxOnRunEffects should receive all actions except STORE_INIT
       expect(logger.actionsLog).toEqual(expectedLog.slice(1));
     });
+
+    it('should dispatch user provided effects actions in order', async () => {
+      let dispatchedActionsLog: string[] = [];
+      TestBed.resetTestingModule();
+      TestBed.configureTestingModule({
+        imports: [
+          StoreModule.forRoot({
+            dispatched: createDispatchedReducer(dispatchedActionsLog),
+          }),
+          EffectsModule.forRoot([
+            EffectLoggerWithOnRunEffects,
+            RootEffectWithInitAction,
+          ]),
+          RouterTestingModule.withRoutes([]),
+        ],
+        providers: [
+          UserProvidedEffect1,
```

Comment:
Yes actually, because they are not provided in the module anymore. Do you have an idea how we could provide the value of a token in the module?

Reply:
Not that I know of 🤔
I would add the provider to the docs tho, instead of only the provider.

```diff
+ providers: [
+ MovieEffects,
  {
    provide: USER_PROVIDED_EFFECTS,
    multi: true,
    useValue: [MovieEffects],
  }
+ }
```

Reply:
Added it